torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -1082,6 +1082,9 @@ struct to_ir {
           throw ErrorReport(stmt)
               << "Unrecognized statement kind " << kindToString(stmt.kind());
       }
+      // Already found an exit statement in this block. Do not parse further
```

Comment:
nit: we've already parsed the remaining block of code, here we're not emitting further because it's not reachable. maybe add a small reference that the remaining statements aren't reachable